### PR TITLE
fix: updated readme with new instructions

### LINKED ISSUE #575: 'Building the Code' steps misleading
**Summary:**

The steps to build the code in README are as misleading as they could be.

**Steps to reproduce:**

Simply read the README :)

**Expected behavior:**
- Fork the repo
- Clone my own fork
- Set upstream to pull changes as and when needed
- No errors observed in Gradle build.

**Observed behavior:**
- Clone the original repo
- Errors will appear in Gradle build

**Device and Android version:**

Any

**Screenshots:**

image

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes #575
Modified readme in the building code section with newer instructions giving contributors an idea about installation of the project.
Please make sure these boxes are checked before submitting your pull request - thanks!

- [.] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [.] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [.] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,19 +54,15 @@ Before you begin, you should have already downloaded the Android Studio SDK and
 
 ## Building the Code
 
-1. Clone the repository using HTTP: git clone https://github.com/openMF/mobile-wallet.git
-
-2. Open Android Studio.
-
+1. Fork the repository.
+2. Go to your fork and clone only the dev branch using `git clone git clone -b <branch> <remote_repo>`
```
Comment (luckyman20) [inline README.md]: Why `git clone` is appearing twice? Also, if you want the developer to clone only dev branch then you should replace `<branch>` with `dev`.

File: README.md [hunk 2]
```diff
@@ -54,19 +54,15 @@ Before you begin, you should have already downloaded the Android Studio SDK and
...
-4. Browse to the directory where you have cloned the mobile-wallet repo and click OK.
-
+4. Browse to the directory where you cloned the mobile-wallet repo and click OK.
 5. Let Android Studio import the project.
-
-6. You will see an error in gradle build related to missing RblClientIdProp and RblClientSecretProp. Add RblClientIdProp and RblClientSecretProp as environment variables with any value.
-
-7. Sync the project again in Android studio.
+6. Let the gradle sync
+7. There should be no errors in gradle/build
```
Comment (luckyman20) [inline README.md]:
```suggestion
7. There should be no errors in gradle build
```

File: README.md (hunk 1 above)
Comment (Bhavnaharitsa) [reply README.md]: I didn't notice the git clone coming twice. Changes done

File: README.md [hunk 3]
```diff
@@ -54,19 +54,15 @@ Before you begin, you should have already downloaded the Android Studio SDK and
...
+3. Click on 'Open an existing Android Studio project'.
+4. Browse to the directory where you cloned the mobile-wallet repo and click OK.
 5. Let Android Studio import the project.
-
-6. You will see an error in gradle build related to missing RblClientIdProp and RblClientSecretProp. Add RblClientIdProp and RblClientSecretProp as environment variables with any value.
-
-7. Sync the project again in Android studio.
+6. Let the gradle sync.
+7. There should be no errors in gradle build.
+8. Set your remote origin and also set upstream to Origin repository to pull changes whenever needed.
```
Comment (jsahil730) [inline README.md]:
```suggestion
8. Set your remote upstream to the remote repository to pull changes whenever needed, using
`git remote add upstream https://github.com/openMF/mobile-wallet.git` 
```
Origin should not be set.

File: README.md [hunk 4]
```diff
@@ -54,19 +54,15 @@ Before you begin, you should have already downloaded the Android Studio SDK and
...
+9. To set the upstream please refer this <a href="https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork">document </a>.
```
Comment (jsahil730) [inline README.md]:
```suggestion
9. Pull changes from dev branch of upstream, whenever needed, using
`git checkout dev`
`git pull upstream dev`
```
You need not do these, I'd prefer mentioning just the commands needed

File: README.md [hunk 5]
```diff
@@ -54,19 +54,15 @@ Before you begin, you should have already downloaded the Android Studio SDK and
...
-1. Clone the repository using HTTP: git clone https://github.com/openMF/mobile-wallet.git
-
-2. Open Android Studio.
-
-3. Click on 'Open an existing Android Studio project'
-
-4. Browse to the directory where you have cloned the mobile-wallet repo and click OK.
-
+1. Fork the repository.
+2. Go to your fork and clone only the dev branch using `git clone -b dev <remote_repo>`.
```
Comment (jsahil730) [inline README.md]: Mention that remote_repo url refers to your own fork and not the 'remote repository'

File: README.md (hunk 3 above)
Comment (Bhavnaharitsa) [reply README.md]: Okay!

File: README.md (hunk 5 above)
Comment (Bhavnaharitsa) [reply README.md]: Okay I shall put that in brackets

File: README.md (hunk 4 above)
Comment (Bhavnaharitsa) [reply README.md]: Okay!

**Pull request closed without merging**